Source/WebCore/platform/ScrollableArea.cpp

```diff
@@ -963,4 +963,9 @@ bool ScrollableArea::shouldBlockScrollPropagation(const FloatSize& biasedDelta)
         && !biasedDelta.width())));
 }
 
+bool ScrollableArea::shouldRegisterScrollbars() const
+{
+    return isListBox() || scrollbarsController().shouldRegisterScrollbars();
```

Comment:
The explicit `isListBox()` check doesn't seem right. Why doesn't its `scrollbarsController()` just provide the correct answer?

Reply:
Fixed.